lib/cc/analyzer/container.rb

```diff
@@ -67,6 +67,13 @@ def run(options = [])
         end
       end
 
+      def stop
+        @on_output = ->(*) { }
```

Comment:
It seems like the process (and output) still runs for a bit even after sending `TERM` or `KILL` so this prevents a bunch of extra errored events from reaching Kafka and triggering unprocessable state changes on the snapshot.